.goreleaser.yml

```diff
@@ -0,0 +1,43 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    id: k8gb 
+    main: ./main.go
+    binary: bin/manager
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+dockers:
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-amd64"
+  use_buildx: false
+  dockerfile: Dockerfile-gr-amd64
+  build_flag_templates:
+  - "--platform=linux/amd64"
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-arm64"
+  use_buildx: false
+  goarch: arm64
+  dockerfile: Dockerfile-gr-arm64
+  build_flag_templates:
+  - "--platform=linux/arm64"
+docker_manifests:
+- name_template: absaoss/k8gb:{{ .Tag }}
+  image_templates:
+  - absaoss/k8gb:{{ .Tag }}-amd64
+  - absaoss/k8gb:{{ .Tag }}-arm64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
```

Comment:
Can you elaborate on the logic here?

Reply:
you can run goreleaser --snapshot, it will build image: image:tag-next and publish it

Reply:
I was wrong on publish: https://goreleaser.com/customization/snapshots/#how-it-works